schema/base.sql

```diff
@@ -2108,6 +2108,11 @@ CREATE TABLE selbaps (
     bapsloc integer NOT NULL
 );
 COMMENT ON TABLE selbaps IS 'Marries selector actions with BAPS servers';
+CREATE TABLE selsources (
```

Comment:
Any reason you're using base.sql rather than patches? Those are easier to apply on prod.

Reply:
oh, we're using them now? Ok :)

Reply:
Done.